### ISSUE TITLE
Error: Cannot find module 'sapper/api.js'

### ISSUE THREAD

**Issue by endel** [sveltejs/sapper-studio#21]
Hey there,

Congrats on this! Looks like a great idea! I've just tried to create a new project and I see this in the logs:

![screen shot 2018-06-05 at 13 07 04](https://user-images.githubusercontent.com/130494/40988454-7c2f9f28-68c1-11e8-9ce4-6e83d66cc29c.png)

**Comment by Rich-Harris** [comment]
ah... yes, that file was added in sapper@0.13, which isn't used by sapper-template. I'd update the template but I believe there might have been some problems with that release (some chatter in Gitter that I haven't had a chance to look into yet) so for now maybe just install `sapper@latest` to the project manually

**Comment by Rich-Harris** [comment]
have fixed the bug in Sapper 0.13 and updated sapper-template, so I'll close this. Thanks!

**Comment by onx2** [comment]
This error has resurfaced for me in **Sapper 0.27.4** and is happening for both the development & production builds.  Maybe I've forgotten a step to get this to work?

On a somewhat related side note, the `npm run sapper` script no longer exists. It seems to have been changed to `npm run build`.

**Steps**: I cloned [Real World Example](https://github.com/sveltejs/realworld), ran `npm i`, then `npm run dev` and was met with a lot of errors from the client and server:

```
> sapper dev

• client
'api.js' is imported by src\routes\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\[slug].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_ArticleMeta.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\editor\[slug].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\editor\_Editor.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_CommentInput.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_Comment.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\register\index.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\settings\index.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\login\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\[view].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\_Profile.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\_components\ArticleList\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\_components\ArticleList\ArticlePreview.svelte, but could not be resolved – treating it as an external dependency
Circular dependency: src\node_modules\@sapper\app.mjs -> src\node_modules\@sapper\internal\App.svelte -> src\routes\_layout.svelte -> src\node_modules\@sapper\app.mjs
Circular dependency: src\node_modules\@sapper\app.mjs -> src\node_modules\@sapper\internal\App.svelte -> src\routes\_layout.svelte -> src\routes\_components\Nav.svelte -> src\node_modules\@sapper\app.mjs
• server
'api.js' is imported by src\routes\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\auth\register.js, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\auth\login.js, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\auth\save.js, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\[slug].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_ArticleMeta.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\editor\[slug].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\editor\_Editor.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_CommentInput.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\article\_Comment.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\register\index.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\settings\index.svelte, but could not be resolved – treating it as an external dependency
'utils.js' is imported by src\routes\login\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\[view].svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\profile\[user]\_Profile.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\_components\ArticleList\index.svelte, but could not be resolved – treating it as an external dependency
'api.js' is imported by src\routes\_components\ArticleList\ArticlePreview.svelte, but could not be resolved – treating it as an external dependency
Circular dependency: src\node_modules\@sapper\app.mjs -> src\node_modules\@sapper\internal\App.svelte -> src\routes\_layout.svelte -> src\node_modules\@sapper\app.mjs
Circular dependency: src\node_modules\@sapper\app.mjs -> src\node_modules\@sapper\internal\App.svelte -> src\routes\_layout.svelte -> src\routes\_components\Nav.svelte -> src\node_modules\@sapper\app.mjs
'del' is imported from external module 'api.js' but never used
✔ service worker (52ms)
internal/modules/cjs/loader.js:651
    throw err;
    ^

Error: Cannot find module 'api.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (C:\Users\Jeff Rooks\Documents\GitHub\sapper-auth\__sapper__\dev\server\server.js:13:11)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
> Server crashed
```
